tests/unit/Util/ExcludeListTest.php

```diff
@@ -17,6 +18,7 @@
 #[CoversClass(ExcludeList::class)]
 #[CoversClass(InvalidDirectoryException::class)]
 #[Small]
+#[RunTestsInSeparateProcesses]
 final class ExcludeListTest extends TestCase
```

Comment:
this test is working with `ExcludeList` which contains a static exclude directories list, see `ExcludeList::addDirectory($directory)`.

this means isolating the test cases makes sure the static state does not trigger side-effects

Reply:
Cherry-picked into `10.5` and merged to `11.0` and `main` from there. I did not cherry-pick the change to `phpunit.xml`, however, as I do not want to run the tests in random order all the time during development.

Reply:
> as I do not want to run the tests in random order all the time during development.

I see. does this mean we can close the remaining PR, or do you plan to run the tests in random order on some particlular event or similar?